Homebrew/README.md

```diff
@@ -28,7 +33,9 @@ You change your path by adding `/usr/local/bin` to your `PATH` environment varia
 This can be done on a per-user basis by adjusting `PATH` in your `~/.bash_profile`.
 To do this, run:
 
-    echo 'PATH="/usr/local/bin:$PATH"' >> ~/.bash_profile
+```console
+$ echo 'PATH="/usr/local/bin:$PATH"' >> ~/.bash_profile
```

Comment:
This `$` is not needed, usually we only include those if we want to separate between running a command and the result.

Reply:
Or to separate multiple commands.

Reply:
right, so i wanted to separate out content which is more of a post install command rather than the actual command needed for the user to use the application. That is the reason i went with the `$` to differentiate the actual instruction commands. Lemme know if you differ with this idea.

Reply:
I did a run through of the entire guide to remove `$` so commands would be easier to copy and run. I think we should stick to that.